releasetasks/__init__.py

```diff
@@ -24,6 +25,13 @@ def make_task_graph(public_key, root_template="release_graph.yml.tmpl",
     now = arrow.now()
     now_ms = now.timestamp * 1000
 
+    signing_pvt_key = None
+    # Don't let the signing pvt key leak into the task graph.
+    if 'signing_pvt_key' in template_kwargs:
+        with open(template_kwargs['signing_pvt_key']) as f:
+            signing_pvt_key = f.read()
+        del template_kwargs['signing_pvt_key']
```

Comment:
Why do you need to delete this member? Maybe pass `signing_pvt_key` (default to `None`) explicitly to be clearer?